service-base/src/main/java/org/eclipse/hono/service/tracing/SamplerProducer.java

```diff
@@ -175,7 +175,7 @@ public Boolean getBoolean(final String name) {
      */
     @Singleton
     @Produces
-    @IfBuildProperty(name = "quarkus.opentelemetry.enabled", stringValue = "true")
+    @IfBuildProperty(name = "quarkus.otel.enabled", stringValue = "true")
```

Comment:
This line should be removed (see other comment).

Reply:
Done.